### ISSUE TITLE
Unable to find localabstract name of cordova app

### ISSUE THREAD

**Issue by lucamorelli** [Microsoft/vscode-cordova#333]
I work to a cordova application using visual studio code under windows.
running the application in an android device using **cordova run android**  from command line works fine, but if I try to run using the vs code task I obtain the error message **Unable to find localabstract name of cordova app**.

this is the output running the command
[run.txt](https://github.com/Microsoft/vscode-cordova/files/1428183/run.txt)

List of devices attached
0123456789ABCDEF
device

adb -s <device_id> shell ps
[shell ps.txt](https://github.com/Microsoft/vscode-cordova/files/1427943/shell.ps.txt)

adb -s <device_id> shell "cat /proc/net/unix"
[shell cat.txt
](https://github.com/Microsoft/vscode-cordova/files/1427946/shell.cat.txt)

Package name: it.dogapp.dogapp

I'm using Visual studio code 1.17.2 and cordva tools 1.2.13

**Comment by victorsosa** [comment]
releated to #215

**Comment by lucamorelli** [comment]
any news about this?

**Comment by iTOYS** [comment]
Hi @lucamorelli,
Unfortunately i can't reproduce this(
Seems like app doesn't start on device("adb shell ps" doesn't contain "it.dogapp.dogapp").
Is is work if you run with console?

**Comment by lucamorelli** [comment]
I rebuilt to project itself to use the cli and now it works, 
I have just a couple of questions. 
**1**. Can I debug the sources using vs code while this task is running? I tried to set breakpoints in the source, but code tells me the breakpoint is ignored.
 this is the task
        ```
{
            "name": "Run Android on device",
            "type": "cordova",
            "request": "launch",
            "platform": "android",
            "target": "device",
            "port": 9222,
            "sourceMaps": true,
            "cwd": "${workspaceRoot}",
            "ionicLiveReload": false
        }
```
**2**. Is it normal that settings ionicLiveReload to true the task fails?

**Comment by iTOYS** [comment]
1. Works fine fo me on `ionic start` project
2. `ionicLiveReload` work only with Ionic1

**Comment by iTOYS** [comment]
Little fix for localabstract name #341

**Comment by iTOYS** [comment]
Fixes #341
